src/components/CountdownTimer/index.js

```diff
@@ -1,19 +1,23 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import classnames from 'classnames'
 
 import classes from './styles.module.scss'
 
-const CountdownTimer = ({ remainingTime }) => {
+const CountdownTimer = ({ remainingTime, extended }) => {
   return (
-    <div className={classes.countdownTimer}>
-      <i className={'far fa-clock'} />
-      <span>{remainingTime}</span>
+    <div className={classnames(classes.countdownTimer, { [classes.extended]: extended })}>
+    {extended
+      ? <><i className={'far fa-clock'} /><span>!Apurate! Esta oferta termina en</span><span>{remainingTime}</span></>
+      : <><i className={'far fa-clock'} /><span>{remainingTime}</span></>
+    }
```

Comment:
I would suggest avoiding repetition:

```suggestion
    <>
      <i className={'far fa-clock'} />
      {extended && <span>!Apurate! Esta oferta termina en</span>}
      <span>{remainingTime}</span>
    </>
```